independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/CreateProjectCommandHandler.java

```diff
@@ -75,6 +90,36 @@ public QuarkusCommandOutcome execute(QuarkusCommandInvocation invocation) throws
         List<Extension> extensionsToAdd = computeRequiredExtensions(invocation.getExtensionsCatalog(), extensionsQuery,
                 invocation.log());
 
+        // Resolves the parent project if it exists. Working for maven projects only
+        final Map<String, Object> parentData = new HashMap<>();
+        final Model parent;
+        final String parentPath = invocation.getStringValue(PROJECT_PARENT_PATH.key());
+
+        switch (invocation.getQuarkusProject().getBuildTool()) {
+            case MAVEN:
+                parent = getMavenParent(parentPath);
+
+                if (parent != null) {
+                    parentData.put(PROJECT_PARENT_GROUP_ID.key(), parent.getGroupId());
+                    parentData.put(PROJECT_PARENT_ARTIFACT_ID.key(), parent.getArtifactId());
+                    parentData.put(PROJECT_PARENT_VERSION.key(), parent.getVersion());
+                    parentData.put(PROJECT_PARENT_PATH.key(), parentPath);
```

Comment:
I think we should log that we detected a parent and therefore we are creating a module.